src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -674,51 +674,43 @@ private bool ConfirmDeployment(Recommendation recommendation)
+            }
+            catch(FailedToCreateDeploymentBundleException ex) when (ex.ErrorCode == DeployToolErrorCode.FailedToCreateContainerDeploymentBundle)
+            {
+                if (_toolInteractiveService.DisableInteractive)
                 {
-                    if (_toolInteractiveService.DisableInteractive)
-                    {
-                        var errorMessage = "Failed to build Docker Image." + Environment.NewLine;
-                        errorMessage += "Docker builds usually fail due to executing them from a working directory that is incompatible with the Dockerfile." + Environment.NewLine;
-                        errorMessage += "Specify a valid Docker execution directory as part of the deployment settings file and try again.";
-                        throw new DockerBuildFailedException(DeployToolErrorCode.DockerBuildFailed, errorMessage);
-                    }
+                    var errorMessage = "Failed to build Docker Image." + Environment.NewLine;
+                    errorMessage += "Docker builds usually fail due to executing them from a working directory that is incompatible with the Dockerfile." + Environment.NewLine;
+                    errorMessage += "Specify a valid Docker execution directory as part of the deployment settings file and try again.";
+                    throw new DockerBuildFailedException(DeployToolErrorCode.DockerBuildFailed, errorMessage);
+                }
 
-                    _toolInteractiveService.WriteLine(string.Empty);
-                    var answer = _consoleUtilities.AskYesNoQuestion("Do you want to go back and modify the current configuration?", "false");
-                    if (answer == YesNo.Yes)
+                _toolInteractiveService.WriteLine(string.Empty);
+                var answer = _consoleUtilities.AskYesNoQuestion("Do you want to go back and modify the current configuration?", "false");
+                if (answer == YesNo.Yes)
+                {
+                    string dockerExecutionDirectory;
+                    do
                     {
-                        var dockerExecutionDirectory =
-                        _consoleUtilities.AskUserForValue(
+                        dockerExecutionDirectory = _consoleUtilities.AskUserForValue(
                             "Enter the docker execution directory where the docker build command will be executed from:",
                             selectedRecommendation.DeploymentBundle.DockerExecutionDirectory,
                             allowEmpty: true);
 
-                        if (!_directoryManager.Exists(dockerExecutionDirectory))
-                            continue;
+                        if (_directoryManager.Exists(dockerExecutionDirectory))
+                        {
+                            _toolInteractiveService.WriteLine($"Error, directory does not exist \"{dockerExecutionDirectory}\"");
+                            break;
+                        }

```

Comment:
I believe this should be the following:
```suggestion
                        if (!_directoryManager.Exists(dockerExecutionDirectory))
                        {
                            _toolInteractiveService.WriteErrorLine($"Error, directory does not exist \"{dockerExecutionDirectory}\"");
                        }

```

Reply:
Done